Sources/Sham/Extensions/StubbedDataCollection+LaunchEnvironmentEncodable.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+extension StubbedDataCollection: LaunchEnvironmentEncodable {
+    /// The key used to save and retrieve the object in the launch environment.
+    public static var launchEnvironmentKey = "mocked-data"
+    // The encoded value of the object in String form, throws an error if unable to encode.
```

Comment:
- Can this use doc comments?
- Can `, throws an error if unable to encode.` be removed and the Throws documentation keyword be used instead?